js/libs/builder.js

```diff
@@ -144,7 +144,8 @@ var app = new Vue({
       showDataSourceSettings: !!formSettings.dataSourceId,
       organizationName: '',
       isPreviewing: formSettings.previewingTemplate !== '',
-      editor: undefined
+      editor: undefined,
+      accessRulesTypes: ['insert']
```

Comment:
what's this for and why doesn't it have "update" too?

Reply:
@squallstar I've added the `accessRulesTypes` variable so we could change its value depending on what setting is set.
In this [file](https://docs.google.com/presentation/d/1vvEvfGm_XGjKE3vp1duMy8DuqrgQGD3Dvlf4M-xNQQU/edit#slide=id.g8d5e20b4f1_0_0) said that we should set the data source rule `update` only if this `Update existing entries in a data source` checkbox is selected. That is why when we click on it we change our variable in what state it should be. You may see it at [this](https://github.com/Fliplet/fliplet-widget-form-builder/pull/330/files#diff-892056c1c88dfac81f33ecf67625a5ecR812) line.